django/core/serializers/json.py

```diff
@@ -62,7 +62,7 @@ def end_object(self, obj):
 
     def getvalue(self):
         # Grand-parent super
-        return super(PythonSerializer, self).getvalue()
+        return super().getvalue()
```

Comment:
Instead of blind search/replace, I am doing a [replace by class name](https://github.com/ChillarAnand/01/blob/master/python/super_exp.py). Here are sample [input](https://github.com/ChillarAnand/01/blob/master/python/super_old.py) & [output](https://github.com/ChillarAnand/01/blob/master/python/super_new.py). Will that work? @aaugustin